### DISCUSSION TITLE
macOS Package Deployment with GLPI-Agent 1.5

### DISCUSSION THREAD

**Discussion by zaheerabbas1988** [glpi-project/glpi-agent#628] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hi everyone,

I'm using GLPI version 10.x with the native inventory module and the GLPI Agent v1.5 installed on my macOS client machines.

My question is:

- Is there a way to deploy macOS packages (e.g., .pkg files) through GLPI?

I understand that GLPI's built-in deployment might be limited to Windows formats like MSI. If that's the case, could you please suggest any recommended approaches for managing macOS package deployments within the GLPI framework? 

Thank you for your assistance!

**Comment by g-bougard** [comment]
Hi @zaheerabbas1988 

there's no limitation for MacOS regarding Deploy task. As far you know how to install a macOS package from the commandline (i.e. using `installer` command), you can install any package. In a Deploy task you'll just have to also setup the command to run to deploy your package.
